securedrop/models.py

```diff
@@ -750,6 +750,9 @@ def to_api_v1(self, all_info: bool = True) -> Dict[str, Any]:
 
         return json_user
 
+    def to_api_v2(self, *args, **kwargs) -> Dict[str, Any]:  # type: ignore
+        return self.to_api_v1(*args, **kwargs)
```

Comment:
I did this lazily on purpose, to demonstrate the case where we want to use the v1 JSON representation verbatim in the v2 API.  I welcome review feedback on whether this is a smell, especially if mypy needs to be told to ignore the passthrough `*args, **kwargs`.  :-)

Reply:
I would prefer we just duplicate the output we want in the v2 function instead of wrapping around v1, to make it easier to understand what's happening and also make it easier for the next person who wants to modify v2 and not v1 (or vice versa).

Re: `all_info=True`, I'm unsure if we want journalists to be able to have access to e.g a list of inactive admin accounts.

I would recommend that we side-step that though and only include the information in v2 that we actually need in the app, and I don't believe we need that right now.

Reply:
Done in 343b3a13ab69dcb4beff7ed86a98447be26c0e4c.